core/sr-std/src/lib.rs

```diff
@@ -45,4 +45,9 @@ pub mod prelude {
 	pub use ::boxed::Box;
```

Comment:
Is there any advantage of using `::boxed::Box` instead of something like `core::boxed::Box` besides typing less?

Reply:
I believe this approach was used because of without_std.rs/with_std.rs trick and we don't know whether `core` is available or not.

Anyway, it is not in the scope of this PR.

Reply:
Hmm yeah I am not sure if I actually prefer having `extern crate std as core;` as a hack to circumvent this problem.

Reply:
If you look into the structure of `sr-std`, you see that we define `boxed` as top-level module. The prelude just re-exports stuff of the top-level module. So, this is exactly what we want.

Reply:
thanks for the explanation!

Reply:
ah yeah that's actually a better explaination!